packages/components/ng-ovh-payment-method/src/payment-method.service.js

```diff
@@ -62,9 +62,10 @@ export default class OvhPaymentMethodService {
       transform: true,
     }).then(
       (paymentMethods) =>
-        find(paymentMethods, {
-          default: true,
-        }) || null,
+        find(
+          paymentMethods,
+          (method) => method.default || method.defaultPaymentMean,
+        ) || null,
```

Comment:
`undefined` wasn't good enough for you ?

Reply:
it's the original behavior i didn't want to change it because it could have some side effects

Reply:
Good point